### ISSUE TITLE
Design matching fund submissions landing and election cycle pages for CMS

### ISSUE THREAD

**Issue by dorothyyeager** [fecgov/fec-cms#3386]
### Summary

**What we are after:**
As an MVP, we have placed the matching funds submission pages into the transition site and added a link to them. We need to move them into the CMS. This work may not need to happen until we prepare for the 2024 election cycle.

### Related issues

- #3251 (Move of these pages from classic.)

### Completion criteria

- [ ] Get background information from Press, Audit and Public Records on these pages and related files (note a 2020 page was created but never went live).
Landing page: https://transition.fec.gov/finance/disclosure/presidential_matching_fund_submissions_2004_present.shtml
https://transition.fec.gov/finance/disclosure/2016MatchingFundSubmissions.shtml
https://transition.fec.gov/finance/2012matching/2012matching.shtml
https://transition.fec.gov/finance/2008matching/2008matching.shtml
https://transition.fec.gov/finance/disclosure/2004MatchingFundSubmissions.shtml

- [x] Decide where those pages and related files will live
- [x] Designs are complete
- [ ] Follow-up implementation tickets exist

### Future work

- Creating pages
- Second pair of eyes
- Publishing pages

**Comment by JonellaCulmer** [comment]
@patphongs @johnnyporkchops @rfultz 
Wondering what you all think about how the process for moving these documents into Wagtail will work. There aren't nearly as many documents to move as with previous migrations. Let me know what you think. 

The reason I ask at this point is because we're thinking through how to engage with the offices that will pick these pages up eventually. I'm assuming our office will move them over and then hand any updates or future pages to the offices that handle this work. 

Those offices include Press, Public Records and Audit.

**Comment by patphongs** [comment]
> Wondering what you all think about how the process for moving these documents into Wagtail will work. There aren't nearly as many documents to move as with previous migrations. Let me know what you think.

There are only a handful of documents for this, so uploading them to Wagtail won't be an issue. We just need to make sure we have redirects in place for when the documents are moved from transition to the Wagtail uploaded content bucket.

**Comment by JonellaCulmer** [comment]
> There are only a handful of documents for this, so uploading them to Wagtail won't be an issue. 

@patphongs So moving them over manually and tracking the links in a spreadsheet for redirects?

**Comment by JonellaCulmer** [comment]
@dorothyyeager Mockups are below. 

**Main landing page:** We need to have a conversation with the other offices referenced in this ticket regarding the introductory language. There will need to be changes to modernize the language and not to repeat information on other pages. In the mockup I've included links in the "Read next" section that take users to that information. I also removed the reference to the sale or use policy and included a link to that page in the "Read next" as well. 

There are two ways we can display the page links. I'm leaning towards the more condensed version of just the year, instead of repeating the same phrase over and over again. 

![Matching fund submissions_mainpage](https://user-images.githubusercontent.com/31663028/114038610-03978780-9850-11eb-9ad8-43e8325025c2.png)

**Election year page:** I've modified the language on this page as well, to an even greater extent. We don't need to repeat the same information on each page. The introduction should reference what's on the page. We also don't need to explain the differences between a PDF and an EXCEL. All that said, I defer to your judgment on the language.

Additionally, as you already know I'd also like to include the sale or use blurb at the bottom of these pages. That text is on the current transition page and I'd like to retain it somehow. 

![2012 submissions page](https://user-images.githubusercontent.com/31663028/114039803-0e065100-9851-11eb-9b7e-d48fe2c6d403.png)

**Comment by JonellaCulmer** [comment]
Two possible development tickets to help create these pages:
- Add document list block to Custom page template
- Add option to use sale or use text on Wagtail pages. You can see this text at the bottom of this page: https://www.fec.gov/data/browse-data/?tab=raising

**Comment by dorothyyeager** [comment]
@JonellaCulmer I agree with you - prefer the version with the non-repeating text. I love your idea of using the sale or use text. And absolutely, custom page template/document list block would be great to have in general.

The intro language definitely needs a rewrite and could cut back on the text that explains the law and simply link to our general information about public funding and matching funds at https://www.fec.gov/introduction-campaign-finance/understanding-ways-support-federal-candidates/presidential-elections/public-funding-presidential-elections/#primary-matching-funds (that might also be a good place to crib from).

**Comment by JonellaCulmer** [comment]
@dorothyyeager Thanks for the review. Do you have time to draft the language for the page, or would you prefer we open a content ticket for next sprint?

**Comment by dorothyyeager** [comment]
Next sprint please @Jonella ***@***.***> - thanks!

**Comment by JonellaCulmer** [comment]
Wrapping up this ticket, we intend to create this pages within:
- Introduction to campaign finance and elections > How to research public records
- Create child pages for the individual years off the new landing page. 

Follow-up tickets:
- https://github.com/fecgov/fec-cms/issues/4541 Draft new language for matching fund submission pages
- https://github.com/fecgov/fec-cms/issues/4542 Add document list block to custom page template
- https://github.com/fecgov/fec-cms/issues/4543 Add option to use sale or use text on Wagtail pages
